packages/sdk/src/types/network/payloads/payloads.ts

```diff
@@ -358,6 +360,17 @@ export type SyncAnimations = Payload & {
     animationStates: SetAnimationState[];
 };
 
+/**
+ * @hidden
+ * App to engine. Starts playing a sound.
+ */
+export type PlaySound = Payload & {

```

Comment:
>PlaySound [](start = 12, length = 9)

A SetSoundState message would be more flexible, allowing you to play/stop, set pitch, etc... all with one message.

Reply:
Agree. I'll have to add a flag for play/pause separate from the flag for stop, but that's fine.